pkg/zrouter/zmiddlewares/logging.go

```diff
@@ -21,7 +21,7 @@ func LoggingMiddleware(next http.Handler) http.Handler {
 		duration := time.Since(start)
 		ctx := r.Context()
 
-		logger.GetLoggerFromContext(ctx).Debugf("Method: %s - URL: %s | Status: %d - Duration: %s - Response Body: %s",
-			r.Method, r.URL.String(), rw.status, duration, rw.Body())
+		logger.GetLoggerFromContext(ctx).Infof("Method: %s - URL: %s | Status: %d - Duration: %s - Response Body: %s",
```

Comment:
I would not log the entire response on info level. We could log everything, but exclude the body. Maybe on debug that can be part of the logging middleware... or maybe something you can enable by config. An answer can be very heavy